src/integrators/type.jl

```diff
@@ -43,6 +43,7 @@ mutable struct DEOptions{absType,relType,QT,tType,F1,F2,F3,F4,F5,F6,tstopsType,d
   force_dtmin::Bool
   advance_to_tstop::Bool
   stop_at_next_tstop::Bool
+  alias_u0::Bool
```

Comment:
I'm not sure this is actually necessary since it's not used for anything.

Reply:
I'll remove it.